source/common/runtime/runtime_features.cc

```diff
@@ -32,8 +32,8 @@
 RUNTIME_GUARD(envoy_reloadable_features_allow_compact_maglev);
 RUNTIME_GUARD(envoy_reloadable_features_append_query_parameters_path_rewriter);
 RUNTIME_GUARD(envoy_reloadable_features_conn_pool_delete_when_idle);
+RUNTIME_GUARD(envoy_reloadable_features_count_mapped_pages_as_free);
```

Comment:
this is a bit ambiguous; maybe `count_unused_mapped_pages_as_free` ?

Reply:
Done